<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingChangeErrorWidget</class>
 <widget class="QWidget" name="SingChangeErrorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="errorLb2">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>23</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(254, 254, 254);</string>
   </property>
   <property name="text">
    <string>&lt;font color= &quot;#22fffe&quot;&gt;阿萨德发射点&lt;/font&gt; 连接异常，将由下一位玩家继续演唱。</string>
   </property>
  </widget>
  <widget class="QLabel" name="waitlooplabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hideBtn">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>101</y>
     <width>91</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover

{
border-image: url(:/image/resourceErrorWidget/bt_off_hover.png);
}
QPushButton
{
border-image: url(:/image/resourceErrorWidget/bt_off.png);
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="errorLb">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>67</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(254, 254, 254);</string>
   </property>
   <property name="text">
    <string>&lt;font color= &quot;#22fffe&quot;&gt;阿萨德发射点&lt;/font&gt; 连接异常，将由下一位玩家继续演唱。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
